backend/battles/forms.py

```diff
@@ -41,32 +41,53 @@ def __init__(self, *args, **kwargs):
         self.fields["opponent"].queryset = User.objects.exclude(email=self.current_user.email)
         self.fields["creator"].initial = self.current_user
 
-    def clean_creator_pokemon_1(self):
+    def clean_creator_pokemon_1_input(self):
         data = self.cleaned_data.get("creator_pokemon_1_input")
         if not pokemon_exists(data):
             self.add_error("creator_pokemon_1_input", "Sorry, this pokemon was not found")
-            return None
-        pokemon = save_pokemon(data)
-        return pokemon
+        return data
 
-    def clean_creator_pokemon_2(self):
+    def clean_creator_pokemon_2_input(self):
         data = self.cleaned_data.get("creator_pokemon_2_input")
         if not pokemon_exists(data):
             self.add_error("creator_pokemon_2_input", "Sorry, this pokemon was not found")
-            return None
-        pokemon = save_pokemon(data)
-        return pokemon
+        return data
 
-    def clean_creator_pokemon_3(self):
+    def clean_creator_pokemon_3_input(self):
         data = self.cleaned_data.get("creator_pokemon_3_input")
         if not pokemon_exists(data):
             self.add_error("creator_pokemon_3_input", "Sorry, this pokemon was not found")
-            return None
-        pokemon = save_pokemon(data)
-        return pokemon
+        return data
 
     def clean(self):
         cleaned_data = super().clean()
         cleaned_data["creator"] = self.fields["creator"].initial
 
+        creator_pokemon_1_input = cleaned_data.get("creator_pokemon_1_input")
+        creator_pokemon_2_input = cleaned_data.get("creator_pokemon_2_input")
+        creator_pokemon_3_input = cleaned_data.get("creator_pokemon_3_input")
+
+        pokemon_points_sum = 0
+        try:
+            pokemon_points_sum = get_pokemons_points_sum(
+                [creator_pokemon_1_input, creator_pokemon_2_input, creator_pokemon_3_input]
+            )
+        except:  # noqa #nosec
```

Comment:
you should always specify the exception. There's only an specific error we want to ignore. By ignoring all exceptions we are allowing for errors that are not expected to pass by silently.

Reply:
Got it. Thank you!